torch/quantization/make_modules.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+_disclaimer = r"""
+# This file is generated using `torch.nn.quantization.make_module`
+# and saved as `{filename}`.
+"""
+
+_imports = r"""
+import torch
+"""
+
+_module_head = r'''
+r"""{module_name} wraps the {op_string} function."""
+class {module_name}(torch.nn.Module):
+'''
+
+_module_init = r'''    def __init__(self, **kwargs):
+        super({module_name}, self).__init__()
+'''
+
+_module_init_extras = r'''
+        self.scale = kwargs.get('scale', 1.0)
+        self.zero_point = swargs.get('zero_point', 0)
+'''
+
+_module_forward = r'''
+    def forward(self, *args):
+        return self.{op_string}(*args)
+'''
+
+_qmodule_forward = r'''
+    def forward(self, *args):
+        return self.{op_string}(
+            *args, scale=self.scale, zero_point=self.zero_point)
+'''
+
+_module_from_float = r'''
+    @classmethod
+    def from_float(cls, mod):
+        assert hasattr(mod, 'observer')
+        qparams = mod.observer.calculate_qparams()
+        return {module_name}(
+            scale=qparams[0].item(), zero_point=qparams[1].item())
+'''
+
+"""Factory method to generate the ops files.
+
+Args:
+    op: The operation to wrap the module around
+"""
+def _make_module(op_string=None, qop_string=None):
+    if op_string is None or qop_string is None:
+        raise ValueError("Both op_string and qop_string must be set.")
+    name = op_string.split('.')[-1].capitalize()
+    name = name + "Gen"
+
+    filename = 'Q' + name + '.py'
```

Comment:
maybe just dump them into a single file? would be easier to import. And probably I wouldn't put it under _generated so that the names are nicer

Reply:
The `_generated` namespace will not be accessible to the user. All the ops will be exposed only through the `quantization` namespace. Or should I put them into their own?

Reply:
Why not have the generated modules show up in two locations:
torch/nn/modules/generated (for float) and torch/nn/quantized/modules/generated (for quantized)?